run_dir/design/qpcr_pools.html

```diff
@@ -0,0 +1,41 @@
+{% extends "base.html" %}
+
+<!--
+Template file: pooling.html
+URL: /qpcr_pools
+Title: List of qPCR pools
+Description: Shows a table with all qPCR pools.
+-->
+
+{% block stuff %}
+
+<div class="container-fluid" id="qpcrpools_list">
+  <div id="pools_table_div">
+      <h1><span id="page_title">qPCR pools</span>
+        <small>Showing all qPCR pools</small>
+      </h1>
+      <table class="table table-striped table-bordered sortable" id="pools_table">
+        <thead>
+          <tr class="sticky">
+            <th class="sort"></th>
+            <th class="sort">Pool</th>
```

Comment:
```suggestion
            <th class="sort w-50">Pool</th>
```

Reply:
Wait, sorry - that's only with Bootstrap 4x 😞 (we should probably upgrade one day..)

```suggestion
            <th class="sort" style="width:50%">Pool</th>
```

Reply:
Made some changes, they should line up now!